packages/nuxt/src/app/components/nuxt-link.ts

```diff
@@ -53,6 +54,32 @@ export function defineNuxtLink (options: NuxtLinkOptions) {
       console.warn(`[${componentName}] \`${main}\` and \`${sub}\` cannot be used together. \`${sub}\` will be ignored.`)
     }
   }
+  const resolveTrailingSlashBehavior = (
+    to: RouteLocationRaw,
+    resolve: (to: RouteLocationRaw) => RouteLocation & { href?: string }
+  ): RouteLocationRaw | RouteLocation => {
+    if (options.trailingSlash !== 'append' && options.trailingSlash !== 'remove') {
+      return to
+    }
+
+    const forceAppend = options.trailingSlash === 'append'
```

Comment:
you can simplify the logic below like this:

```suggestion
    const method = options.trailingSlash === 'append' ? withTrailingSlash : withoutTrailingSlash
```